webpack/webpack.dev.babel.js

```diff
@@ -0,0 +1,108 @@
+import ExtractTextPlugin from 'extract-text-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import StringReplacePlugin from 'string-replace-webpack-plugin';
+import WebpackNotifierPlugin from 'webpack-notifier';
+
+import packageInfo from '../package';
+import webpackConfig from './webpack.config.babel';
+
+// Defaults to value in package.json.
+// Can override with npm config set dcos-ui:port 80
+let PORT = process.env.npm_package_config_port;
+let proxy;
+
+try {
+  // If this fails - you should create a proxy.dev.js by copying proxy.template.js
+  // and customizing the proxy as needed.
+  proxy = require('./proxy.dev.js');
```

Comment:
Again here, I think it's ok to fail if the file doesn't exist.
Maybe we can have an npm script that just copies this proxy file to `.dev` and same thing for the `config.template` that way it's easy for someone to get started.